packages/react/src/Textarea/Textarea.docs.json

```
@@ -60,7 +60,13 @@
     {
       "name": "sx",
       "type": "SystemStyleObject"
+    },
+    {
+      "name": "className",
+      "type": "string | undefined",
+      "defaultValue": "",
+      "description": "CSS string"
```

```suggestion
      "description": "The className to apply to the wrapper element"
```

Just a suggestion based on the TS description 👀